test/common/testvm.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.arch = "x86_64"
 
         self.image = image or testinfra.DEFAULT_IMAGE
+        self.atomic_image = self.image in ["rhel-atomic", "fedora-atomic", "continuous-atomic"]
```

Comment:
What is the reason for this change? It barely saves any code in the tests and adds an extra place where we need maintenance when the list of images changes.